watertap/data/techno_economic/injection_well_disposal.yaml

```diff
@@ -2,29 +2,21 @@ default:
   energy_electric_flow_vol_inlet:
     value: 0.409594654
     units: kWh/m^3
-  specific_capital_cost_flow_vol_inlet:
-    flow_basis_par:
+  capital_cost:
+    basis: flow_vol
+    cost_factor: None
+    reference_state:
       value: 473.176473
       units: m^3/hr
-    cap_basis_par:
+    capital_a_parameter:
       value: 22.5
-      units: M$
-    cap_exp_par:
+      units: MUSD_2011
```

Comment:
Is there any concern that we use MUSD as the currency unit here?

Reply:
Whilst it doesn't matter in the end, it would probably be a good idea to be consistent with what we use here.

Reply:
Purely convenience! Conversion to proper units is handled elsewhere I believe.